### ISSUE TITLE
Unit test: Jest looking for babel configuration

### ISSUE THREAD

**Issue by hdurix** [jhipster/jhipster-vuejs#152]
When running the unit tests in a generated app, we have some warning about babel not found (and ignored) with Jest:

```
[vue-jest]: no .babelrc found, skipping babel compilation
```

There is maybe an option to explain to Jest that babel is not used.

**Comment by deepu105** [comment]
That's weird, we are using Jest on React and Angular as well without Babel, of course it internally uses whatever it needs but does not make it required at project level. Also its just a normal log I wouldn't bother about it at all

**Comment by sahbi-ktifa** [comment]
Yes, we do not require it as top level dependency but that's the reason of the warning :)

**Comment by deepu105** [comment]
lets ignore it then, its not worth it

Thanks & Regards,
Deepu
